src/components/search-result-list/search-result-list-item/search-result-list-item.tsx

```diff
@@ -34,17 +35,21 @@ const SearchResultListItem: React.FC<SearchResultListItemProps> = ({
   coverTint
 }) => {
   const t = useText();
+  const { materialUrl } = useUrls();
   const creatorsText = creatorsToString(
     flattenCreators(filterCreators(creators, ["Person"])),
     t
   );
   const author = creatorsText || "[Creators are missing]";
   const datePublished = getFirstPublishedYear(manifestations);
   const manifestationPid = getManifestationPid(manifestations);
+  const firstInSeries = series?.[0];
+  const { title: seriesTitle, numberInSeries } = firstInSeries;
+  const materialFullPath = `${materialUrl}/${workId}`;
```

Comment:
Consider adding support for placeholders in urls.

We have previously worked with placeholders for values in props. If the material url from `useUrls()` is `/work/:workId` then we replace `:workId` with the actual value. This reduces the coupling between routing in the original application

Reply:
yep